build-tools/packages/build-cli/src/args.ts

```diff
@@ -39,6 +39,16 @@ export const findPackageOrReleaseGroup = (
 	);
 };
 
+/**
+ * Creates a CLI argument for release group names. It's a factory function so that commands can override the
+ * properties more easily when using the argument.
+ */
+export const releaseGroupArg = Args.custom({
```

Comment:
```suggestion
export const releaseGroupArg = Args.custom<ReleaseGroupName>({
```

Also add a parse function that returns a ReleaseGroupName. See the semverArg for an example.